sound/soc/sof/ipc4-topology.c

```diff
@@ -692,6 +693,7 @@ static int sof_ipc4_widget_setup_comp_pipeline(struct snd_sof_widget *swidget)
 	}
 
 	swidget->core = pipeline->core_id;
+	spipe->core_mask |= BIT(pipeline->core_id);
```

Comment:
Do we need to clear core_mask?

Reply:
This is set during topology parsing and will remain unchanged until the topology is unloaded. I don't think we need to clear the core mask

Reply:
it will get reset anyway because all IPC structures are reallocated